.circleci/config.yml

```diff
@@ -35,14 +38,21 @@ commands:
     description: 'Save the Gradle cache'
     steps:
       - save_cache:
-          key: 'v1-gradle-dependencies-{{ arch }}-{{ checksum "android/build.gradle" }}-{{ checksum "android/app/build.gradle" }}-{{ checksum "node_modules/react-native/package.json" }}'
+          key: &gradle-key >-
```

Comment:
I'll assume that since Circle didn't barf when it got this that it will actually handle it correctly. Please change to `gradle-cache-key`.